SampleServer.cpp

```diff
@@ -1,3 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2019-2021 (c) Christian von Arnim, ISW University of Stuttgart  (for VDW and umati)
+ *    Copyright 2020 (c) Tonja Heinemann, ISW University of Stuttgart  (for VDW and umati)
+ *    Copyright 2021 (c) Goetz Goerisch, VDW - Verein Deutscher Werkzeugmaschinenfabriken e.V.
+ */
+
+#include "src_generated/namespace_di_generated.h"
+#include "src_generated/namespace_ia_generated.h"
+#include "src_generated/namespace_machinery_generated.h"
+#include "src_generated/namespace_machinetool_generated.h"
+/*#include "src_generated/namespace_robotics_generated.h"*/
```

Comment:
Linter forces a specific ordering of includes, the "internal" ones must be included lastly.

Reply:
fixed